src/Lykke.AlgoStore.Core/Settings/SlackNotifications/SlackNotificationsSettings.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.AlgoStore.Core.Settings.SlackNotifications
+{
+    public class SlackNotificationsSettings
+    {
+        public AzureQueuePublicationSettings AzureQueue { get; set; }
```

Comment:
You can use Lykke.AzureQueueIntegration.AzureQueueSettings instead

Reply:
Latest code in lykke.dotnettemplates still uses AzureQueuePublicationSettings so I am not sure what to do here. Can you, please, provide me with link to some of existing repos that have this already implemented and I will take a look and update code accordingly?
Thanks

Reply:
For example https://github.com/LykkeCity/Lykke.Service.Regulation/blob/master/src/Lykke.Service.Regulation/Settings/SlackNotifications/SlackNotificationsSettings.cs
https://github.com/LykkeCity/Lykke.Service.Regulation/blob/master/src/Lykke.Service.Regulation/Startup.cs#L191
It's just an advice, not critical issue.